region/crateria/central.json

```diff
@@ -3210,6 +3223,18 @@
                   "name": "Base",
                   "notable": false,
                   "requires": [ "h_canBombThings" ]
+                },
+                {
+                  "name": "G-mode Morph to Bomb and overload Plms",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Bombs"
+                  ]
```

Comment:
Assuming I'm understanding this strat, maybe add a note that this is bombing the crumble to overload plms, then going through it.